custom_components/tahoma/tahoma_device.py

```diff
@@ -170,6 +170,9 @@ async def async_execute_command(self, command_name: str, *args: Any):
             _LOGGER.error(exception)
             return
 
+        if self.assumed_state:
```

Comment:
I only need to see if this doesn't break the stop functionality. Not sure if the executions are currently cancelled to stop RTS devices.

Reply:
Good point. @vlebourl You have RTS covers right?

Reply:
@tetienne and maybe we should also add logic in `is_opening` and `is_closing` in cover.py to only do this for IO devices! (another PR)

Reply:
I don’t think so. The code is already protected as we check the execution list. Except if you are concerned about performance?

Reply:
@tetienne I was indeed wondering about performance, but I thought it was doing an API call. It is just retrieving the values from the local store, which is fine to keep like this.

Reply:
So now we have only to know if it breaks the stop feature for RTS devices right?

Reply:
@iMicknl A possible solution would be to store the exec_id directly into the device as a variable. 

```Python
async def async_execute_command(self, command_name: str, *args: Any):
        """Execute device command in async context."""
        try:
            self.last_exec_id= await self.coordinator.client.execute_command(
                self.device.deviceurl,
                Command(command_name, list(args)),
                "Home Assistant",
            )
        except Exception as exception:  # pylint: disable=broad-except
            _LOGGER.error(exception)
            return

        if self.assumed_state:
            return

        # ExecutionRegisteredEvent doesn't contain the deviceurl, thus we need to register it here
        self.coordinator.executions[self.last_exec_id] = {
            "deviceurl": self.device.deviceurl,
            "command_name": command_name,
        }

        await self.coordinator.async_refresh()
```